var/spack/repos/builtin/packages/nwchem/package.py

```diff
@@ -115,7 +115,7 @@ def install(self, spec, prefix):
 
         if "+fftw3" in spec:
             args.extend(["USE_FFTW3=y"])
-            args.extend(["LIBFFTW3=%s" % fftw.ld_flags])
+            args.extend(["LIBFFTW3=%s" % fftw.joined(" ")])
```

Comment:
Can you explain what was the error before, which line got added and which one is being used now?